src/deps_info.json

```diff
@@ -40,6 +40,7 @@
   "gai_strerror": ["malloc"],
   "setprotoent": ["malloc"],
   "emscripten_run_script_string": ["malloc", "free"],
+  "emscripten_log": ["strlen"],
```

Comment:
This is not correct - `emscripten_log` does not depend on `strlen`? `_formatString` does, and it does already have an entry in here:
```
  "_formatString": ["strlen"],
```

Reply:
(This code was fixed/changed recently here: https://github.com/emscripten-core/emscripten/pull/7672/files#diff-58c5cfd7d0915fd265929f9ea50fc0e9R71)

Reply:
Hmm, I believe this is how `deps_info.json` works: as we link the object files from the user, in order to link in system libraries, we see X is required. If we know that X will come from JS actually, and not the C system libraries, and X requires Y from a system library, then we must link in Y from system libraries now - because otherwise we will only see that we need Y later in JS linking, which is too late for more C code.

So we can't see `_formatString` at this stage. It's a dependency of a JS item, which is a relationship only the JS linker is aware of. Instead, we see that `emscripten_log` is used, and we need to infer that JS linking will later require `strlen`.

So I think this change is correct, but we should also remove `_formatString` from this file, unless I'm missing something?